e2e/test/MessageSendE2ETests.cs

```diff
@@ -80,7 +81,8 @@ public async Task Message_DeviceSendSingleMessage_AmqpWs_WithHeartbeats()
         }
 
         [TestMethod]
-        [TestCategory("ProxyE2ETests")]
+        [TestCategory("Proxy")]
+        [TestCategory("LongRunning")]
```

Comment:
This takes long?

Reply:
I've been trimming away anything that takes more than 10 seconds